src/components/ActionCard.tsx

```diff
@@ -89,15 +89,16 @@ const ActionCard: React.FC<IProps> = ({
   isRight,
   isBottom = true,
 }) => {
-  const isImageURL = typeof image === "string" && image.includes("http")
+  const isImageURL = typeof image === "string"
```

Comment:
Simplifying this condition. Any sort of string must be passed to an html `img` tag.